gnomad_qc/v4/annotations/generate_freq_genomes.py

```diff
@@ -97,6 +93,39 @@
 FREQ_GLOBALS = ("freq_meta", "freq_meta_sample_count")
 """Global annotations on the frequency Table."""
 
+DOWNSAMPLINGS = [
```

Comment:
Just so I know, where did this list come from? Was it from the freq_meta?

Reply:
Yes, I got the from get_freq() for genomes and then freq_meta.collect(), Mike also pasted in our Slack thread.